s3/compare-credentials-jumpbox.sh

```diff
@@ -0,0 +1,17 @@
+# Pass the credential file as an argument
```

Comment:
Rename this one to `compare-credentials.sh` and delete the other one

Reply:
done